tests/test_route_task.py

```
@@ -0,0 +1,188 @@
+"""‏`routing.order` אוכף בפועל — לא מוצהר.
+
```

Ruff (0.16.3)

[error] Contains control characters that can permit obfuscated code

(PLE2502)